docs/content/direct/windows-setup-guide.md

```diff
@@ -0,0 +1,134 @@
+
+##  KubeStellar Installation Guide for Windows (PowerShell/CMD)
+
+## Work in Progress -- More steps coming soon!!!!!!
+
+### Prerequisites
+
+You need to install the following tools:
+
+1. [Git for Windows](https://git-scm.com/)
+2. [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+3. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/)
+4. [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+5. [Helm](https://helm.sh/docs/intro/install/)
+6. [OCM CLI (`clusteradm`)](https://open-cluster-management.io/)
+7. [kubeflex CLI (`kflex`)](https://github.com/kubestellar/kubeflex)
+
+
+
+###  Step-by-Step Windows Installation
+
+### Invoke-WebRequest -> This is a built-in PowerShell command used to download files or access web pages. It's similar to curl or wget in Linux/macOS
+
+### -Uri -> This specifies the URL you want to request/download from
+
+### -OutFile -> This specifies where to save the downloaded file on your computer
+
+### "$env:USERPROFILE\kind.exe"  -> This uses an environment variable $env:USERPROFILE to refer to your Windows user folder (like C:\Users\YourName), and saves the file there as kind.exe
+
+
+
+#### 1. Install kubectl
+
+```powershell
+Invoke-WebRequest -Uri "https://dl.k8s.io/release/v1.32.2/bin/windows/amd64/kubectl.exe" -OutFile "$env:USERPROFILE\kubectl.exe"
+```
+
```

Comment:
The rendering does not properly identify the textual structure here.

<img width="719" alt="image" src="https://github.com/user-attachments/assets/6225f735-cd2f-4f4a-894e-33d6d42d2273" />

The actual structure is three steps --- fetch, add to PATH, and test. The "add to PATH" step consists of two sub-steps. Indenting with _four_ spaces is recognized by MkDocs and by GitHub markdown as indicating a nested paragraph.